### PR TITLE
Responsive Portfolio Website for desktop Devices

### PR DESCRIPTION
**Pull request by David-Lanzz**
. Change background images for main content and contact cards
. Make the website responsive when used on desktop and mobile devices
. Add a new development link for live demo to the README file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,30 +76,31 @@ <h3>See Project</h3>
...
+                            <li class="bed">Lead Developer</li>
+                            <li class="year">2018</li>
                         </ul>
                     </div>
                 </li>
                 <li class="aboutProject">
-                    A daily selection of privately <br> personalized reads: no accounts or <br> sign-ups required.
-                </li>
+                    A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive
+                    computer: your car. </li>
```
Comment (wayungi) [inline index.html]: - [OPTIONAL]  This is supposed to be a paragraph. `li` tags are used to present items that are part of a list or navigation menu. What you have done here is semantically incorrect and I suggest that you change it.

File: styles.css [hunk 2]
```diff
@@ -560,17 +531,192 @@ textarea {
...
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .aboutme {
+    display: grid;
+    grid-template-rows: 3fr 3fr;
```
Comment (wayungi) [inline styles.css]: - [ ] Please note that in the design, the left section is 40% of the width and the right side is 60% of the width. You have made them both 50%. Please change this to match the design.

_**figma**_
figma-1

_**Your desgn**_
figma-2

File: index.html [hunk 3]
```diff
@@ -219,14 +229,15 @@ <h2>HTML</h2>
...
+        </div>
             <li class="frameInfo">Frameworks
                 <p>></p>
             </li>
             <li class="frameInfo">Skills
                 <p>></p>
             </li>
         </ul>
-        </ul>
+
```
Comment (wayungi) [inline index.html]: - [ ] Please make the language, skills, and framework sections to be dropdown menus. Take a look at this [article](https://www.freecodecamp.org/news/html-drop-down-menu-how-to-add-a-drop-down-list-with-the-select-element/) from free code camp to learn about drop-down menus

File: index.html [hunk 4]
```diff
@@ -36,7 +41,7 @@ <h1>I'm David <br>Glad to see you!</h1>
     </section>
     <section class="works">
```
Comment (wayungi) [inline index.html:42]: - [ OPTIONAL] If you look carefully, you will realize that the ordering of the projects on the mobile and desktop version are different. This is not a bug but a feature. You can try hitting that mark with the flexbox property **order**. Read more about it [here](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)

File: index.html [hunk 5]
```diff
@@ -143,30 +150,31 @@ <h3>See Project</h3>
...
                             <li class="year">2015</li>
                         </ul>
                     </div>
                 </li>
                 <li class="aboutProject">
-                    A daily selection of privately <br> personalized reads: no accounts or <br> sign-ups required.
-                </li>
+                    Experimental content creation feature that allows users to add to an existing story over the course
+                    of a day without spamming their friends. </li>
                 <li class="lang">
```
Comment (wayungi) [inline index.html]: - [ ] Please add a margin-left  `li here to achieve the correct spacing. Your spacing is too small

- [OPTIONAL] Note that this `li` tag  should be a `p` tag in order to retain its semantic meaning
_**figma**_
space-1

_**your design**_
space-2

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @David-Lanzz  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## Highlights 💯 
- [ ] Good PR title ✔️ 
- [ ] Descriptive commit message ✔️ 
- [ ] Profeesional README ✔️ 
- [ ] All linter checks passing ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -199,11 +207,13 @@ <h3>
...
         <ul class="langInfo">
-            <h3 langs>Languages
+            
+            <h3 class = "langs" id="langs">Languages
                 <p class="v">v</p>
             </h3>
+            <div class="plangsContainer">
             <li class="pLangs javaScript">
                 <img class="image" src="./images/Ellipse 1 (1).png"></img>
                 <h2>JavaScript</h2>
```
Comment (IngDavidTM) [inline index.html]: - [ ] While you did a wonderful job styling your project :+1: , as requested before you should make the language, skills, and framework sections to be dropdown menus and add 5 tags per row listing the technologies :+1:. You can check this [video](https://www.youtube.com/watch?v=eKo1NV1qxbc) where it is done using only HTML and CSS.

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @David-Lanzz  🙋‍♂️, 

Good job so far!
You have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @David-Lanzz  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**